apps/site/pages/en/learn/userland-migrations/introduction.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Userland Migrations
+layout: learn
+authors: JakobJingleheimer, AugustinMauroy
+---
+
+![Node.js Userland Migrations](https://raw.githubusercontent.com/nodejs/userland-migrations/main/.github/assets/Userland-Migration-Tagline.png)
+
+# Userland Migrations
+
+Node.js provides migrations for "userland" (what you write vs node's own) source-code to facilitate adoption of new features and upgrading source-code affected by breaking changes. These are done in collaboration with [`codemod`](https://docs.codemod.com/introduction), who also work with other major projects like Next.js, React, and Tailwind. Node.js's migrations live in the [`nodejs/userland-migrations`](https://github.com/nodejs/userland-migrations) repository and are overseen by the `@nodejs/userland-migrations` team.
```

Comment:
```suggestion
Node.js provides migrations for "userland"(anything not included in the `node` executable itself) to facilitate the adoption of new features and breaking changes. These are done in collaboration with [Codemod](https://docs.codemod.com/introduction), a company that specializes in xyz.

For Node.js, migrations live in https://github.com/nodejs/userland-migrations, and are overseen by the project itself.
```

Reply:
some alternative:

Node.js offers migrations for "userland" code (anything outside the `node` executable) to help adopt new features and handle breaking changes. These are built in collaboration with [Codemod](https://docs.codemod.com/introduction), a company focused on making it easy to build, share, and run codemods.

Reply:
I'm not fan of term company (we previously had issue with theses kind of thing). Maybe use term platform because we (userland team) only use registry & studio